providers/curation/github.js

```diff
@@ -185,6 +201,50 @@ class GitHubCurationService {
     }
   }
 
+  /**
+   * Given a partial spec, return the list of full spec urls for each curated version of the spec'd components 
+   * @param {string} searchPattern - a partial path that describes the sort of curation to look for. 
+   * @returns {[URL]} - Array of URLs describing he available curations
+   */
+  async search(searchPattern) {
+    await this.ensureCurations();
+    const root = `${this.tempLocation.name}/${this.options.repo}/${this._getSearchRoot(searchPattern)}`;
+    if (!fs.existsSync(root))
+      return [];
```

Comment:
Since it's an Express app, any sync functions should not be used  here because it would freeze the Node loop. fs.exists is deprecated, so something else should be used instead...
I would also add curly braces here to avoid any possible bugs in the future when another line is added etc.